lava/chain.json

```diff
@@ -0,0 +1,119 @@
+{
+    "$schema": "../chain.schema.json",
+    "chain_name": "lava",
+    "status": "upcoming",
+    "network_type": "mainnet",
+    "website": "https://lavanet.xyz/",
+    "update_link": "https://raw.githubusercontent.com/lavanet/lava/main/chain.schema.json",
+    "pretty_name": "Lava",
+    "chain_id": "lava-mainnet-1",
+    "bech32_prefix": "lava@",
+    "daemon_name": "lavad",
+    "node_home": "$HOME/.lava",
+    "key_algos": [
+        "secp256k1"
+    ],
+    "slip44": 118,
+    "fees": {
+        "fee_tokens": [
+            {
+                "denom": "ulava",
+                "fixed_min_gas_price": 0.000000001,
+                "low_gas_price": 0.000000001,
+                "average_gas_price": 0.00005,
+                "high_gas_price": 0.0001
+            }
+        ]
+    },
+    "staking": {
+        "staking_tokens": [
+            {
+                "denom": "ulava"
+            }
+        ],
+        "lock_duration": {
+            "time": "1814400s"
+        }
+    },
+    "codebase": {
+        "git_repo": "https://github.com/lavanet/lava",
+        "recommended_version": "v0.33.1",
+        "compatible_versions": [
+            "v0.33.1"
+        ],
+        "binaries": {
+            "linux/amd64": "https://github.com/lavanet/lava/releases/download/v0.33.1/lavad-v0.33.1-linux-amd64?checksum=sha256:ae2e49e53bd8c979ca27c83d3d10a708fc23247dae020840fe46b9b68cb8e925"
+        },
+        "go_version": "1.20",
+        "cosmos_sdk_version": "github.com/lavanet/cosmos-sdk@v0.47.x-lava",
+        "consensus": {
+            "type": "cometbft",
+            "version": "0.37.4"
+        },
+        "cosmwasm_enabled": false,
+        "ibc_go_version": "v7.2.0",
+        "ics_enabled": [
+            "ics20-1"
+        ],
+        "versions": [
```

Comment:
fyi, when using a versions.json file, the versions array is not required. (In fact, I recommend against it because it means two places to update and can lead to de-sync). But it's not a strict rule at this point.